home_work_3/seminar_task_6.py

```diff
@@ -0,0 +1,15 @@
+# Задание №6 семинара 3. Работа с выводом строки.
+
+my_str = input('Введите строку: ')
+my_str = my_str.split()
+my_str = sorted(my_str)
+len_max = 0
+counter = 1
+
+for item in my_str:
+    if len(item) > len_max:
+        len_max = len(item)
+
+for item in my_str:
+    print(str(counter), ' ', (len_max - len(item)) * ' ', item, sep='')
+    counter += 1
```

Comment:
Можно проще, используя форматирование
```suggestion
for i, world in enumerate(my_str, start=1):
    print(f'{i} {world:>{len_max}}')
```